mongorestore/mongorestore.go

```diff
@@ -34,6 +34,12 @@ const (
 	progressBarWaitTime = time.Second * 3
 )
 
+const (
+	deprecated_db_and_collection_options_warning = "The --db and --collection flags are deprecated for " +
```

Comment:
[nit] golang typically does not use snake_case
```suggestion
	deprecatedDbAndCollectionOptionsWarning = "The --db and --collection flags are deprecated for " +
```